cmd/minishift/cmd/start.go

```diff
@@ -419,6 +425,11 @@ func initClusterUpFlags() *flag.FlagSet {
 	clusterUpFlagSet.AddFlag(cmdutil.HttpProxyFlag)
 	clusterUpFlagSet.AddFlag(cmdutil.HttpsProxyFlag)
 
+	if future {
```

Comment:
and this use seems to be clearer in intention.
not sure how hidden is treated in documentation. in this case it does not show up at all...